src/main/java/az/edu/turing/controller/BookingController.java

```diff
@@ -0,0 +1,32 @@
+package az.edu.turing.controller;
+
+import az.edu.turing.model.dto.BookingDto;
+import az.edu.turing.service.BookingService;
+import az.edu.turing.service.impl.BookingServiceImpl;
+
+import java.util.List;
+import java.util.Optional;
+
+public class BookingController {
+    private BookingService bookingService = new BookingServiceImpl();
+
+    public BookingDto createBooking(long flightId, String firstName, String lastName) {
+        return bookingService.createBooking(flightId, firstName, lastName);
+    }
+
+    public boolean cancelBooking(long bookingId) {
+        return bookingService.cancelBooking(bookingId);
+    }
+
+    public List<BookingDto> findBookingByPassenger(String fullName) {
```

Comment:
1 passenger can book multiple flights. It would be better to name method 'findAllBookingsByPassenger'